### ISSUE TITLE
[Question] Version Number via Script

### ISSUE THREAD

**Issue by PS3-Pro** [aldostools/webMAN-MOD#746]
Aldo is there a way to the script to check the FW version w

```
If Firmware 4.89
.
.
.
Else if Firmware 4.88
,
,
,
else if Firmware 4.84
```

**Comment by aldostools** [comment]
Not currently.

**Comment by PS3-Pro** [comment]
> Not currently.

NP, I was just thinking of a way to do this automatically, I made a generic layout_grid.txt file to work on both firmwares (4.89 and below) without too many visual bugs using 4.88 rcos and an option for the user to select the proper fix for 4.88 ( original and bug free) and 4.89 ( 4.88 layout with some values of 4.89 from the web browser to not break the mouse pointer, but in exchange for moving notifications down a bit)

**Comment by aldostools** [comment]
Lasted build now supports it
